cmd/example-mysql/main.go

```diff
@@ -87,3 +119,33 @@ func main() {
 		klog.Exitf("ListenAndServe: %v", err)
 	}
 }
+
+// parseTileLevelIndex takes level and index in string, validates and returns them in uint64.
+//
+// Examples:
+// "/tile/0/x001/x234/067" means level 0 and index 1234067 of a full tile.
+// "/tile/0/x001/x234/067.p/8" means level 0, index 1234067 and width 8 of a partial tile.
```

Comment:
Add a comment that this doesn't return any of the partial tile component?

Reply:
Done.